jobs/kubelet/templates/config/gce.gcfg.erb

```diff
@@ -0,0 +1,5 @@
+<% if p("cloud-provider.type") == "gce" %> 
```

Comment:
Could this file name be more generic and used for the other providers? eg `cloud.cfg`